tgui/packages/tgui/interfaces/CrewConsoleSkyrat.js

```diff
@@ -0,0 +1,198 @@
+
+const jobToColor = (jobId) => {
+  if (jobId === 0) {
+    return COLORS.department.captain;
+  }
+  if (jobId >= 10 && jobId < 20) {
+    return COLORS.department.security;
+  }
+  if (jobId >= 20 && jobId < 30) {
+    return COLORS.department.medbay;
+  }
+  if (jobId >= 30 && jobId < 40) {
+    return COLORS.department.science;
+  }
+  if (jobId >= 40 && jobId < 50) {
+    return COLORS.department.engineering;
+  }
+  if (jobId >= 50 && jobId < 60) {
+    return COLORS.department.cargo;
+  }
+  if (jobId >= 60 && jobId < 80) {
+    return COLORS.department.service;
+  }
+  if (jobId >= 200 && jobId < 240) {
+    return COLORS.department.centcom;
+  }
+  return COLORS.department.other;
+};
+
+const healthToAttribute = (oxy, tox, burn, brute, attributeList) => {
+  const healthSum = oxy + tox + burn + brute;
+  const level = Math.min(Math.max(Math.ceil(healthSum / 50), 0), 5);
+  // 200 Default Health, Sum Divided by 50, 6 Health States
+  return attributeList[level];
+};
+
+const HealthStat = (props) => {
+  const { type, value } = props;
+  return (
+    <Box inline width={2} color={COLORS.damageType[type]} textAlign="center">
+      {value}
+    </Box>
+  );
+};
+
+export const CrewConsoleSkyrat = () => {
+  return (
+    <Window title="Crew Monitor" width={600} height={600}>
+      <Window.Content scrollable>
+        <Section minHeight="540px">
+          <CrewTable />
+        </Section>
+      </Window.Content>
+    </Window>
+  );
+};
+
+const CrewTable = (props, context) => {
+  const { act, data } = useBackend(context);
+  const sensors = sortBy((s) => s.ijob)(data.sensors ?? []);
+  return (
+    <Table cellpadding="3">
+      <Table.Row>
+        <Table.Cell bold colspan="2">
+          Name
+        </Table.Cell>
+        <Table.Cell bold collapsing textAlign="center">
+          Status
+        </Table.Cell>
+        <Table.Cell bold collapsing textAlign="center">
+          Vitals
+        </Table.Cell>
+        <Table.Cell bold width="180px" collapsing textAlign="center">
+          Position
+        </Table.Cell>
+      </Table.Row>
+      {sensors.map((sensor) => (
+        <CrewTableEntry sensor_data={sensor} key={sensor.ref} />
+      ))}
+    </Table>
+  );
+};
+
+const CrewTableEntry = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { link_allowed } = data;
+  const { sensor_data } = props;
+  const {
+    name,
+    assignment,
+    ijob,
+    is_robot,
+    life_status,
+    oxydam,
+    toxdam,
+    burndam,
+    brutedam,
+    area,
+    can_track,
+  } = sensor_data;
+
+  return (
+    <Table.Row>
+      <Table.Cell bold={jobIsHead(ijob)} color={jobToColor(ijob)}>
+        {name}
+        {assignment !== undefined ? ` (${assignment})` : ''}
+      </Table.Cell>
+      <Table.Cell collapsing textAlign="center">
+        {is_robot ? <Icon name="wrench" color="#B7410E" size={1} /> : ''}
+      </Table.Cell>
+      <Table.Cell collapsing textAlign="center">
+        {oxydam !== undefined ? (
```

Comment:
This should make it so it displays a skull icon when they're dead, even if their health is all fine and dandy. Do make sure to test it before committing it.
```suggestion
        {oxydam !== undefined && life_status ? (
```